winterfell/middlewares/expressLimiter.js

```diff
@@ -0,0 +1,47 @@
+var client = require('redis').createClient();
+var expressLimiter = require('express-limiter');
+var Constants = require('../utils/constants');
+var Log = require('./log');
+
+/**
+ * Adds a limiter if we are running in prod which globally
+ * limits requests to 360 per hour.
+ *
+ * @param app Express app.
+ */
+function addLimiter(app) {
+  if (app.environment === Constants.environment.PROD) {
+    var limiter = expressLimiter(app, client);
+
+    limiter({
+      path: '/',
+      method: 'all',
+      lookup: ['connection.remoteAddress'],
+      // 360 request per hour per address for all pages
```

Comment:
is 360/hr standard? I feel like it should be within a smaller window.
Attacks usually happen when a lot more requests are sent within a few minutes.